ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessRewriter.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.restmodel.displayedgraph
+
+import pl.touk.nussknacker.engine.graph.node.{Disableable, NodeData}
+import pl.touk.nussknacker.restmodel.displayedgraph.rewriter.{RemoveAndBypassNode, RemoveBranch}
+
+object ProcessRewriter {
+
+  def removeAndBypassNode(process: DisplayableProcess, node: NodeData): DisplayableProcess =
+    new RemoveAndBypassNode(process, node).apply
+
+  def removeBranch(process: DisplayableProcess, branchRoot: NodeData): DisplayableProcess =
+    new RemoveBranch(process, branchRoot).apply
+
+  def removeDisabledNodes(process: DisplayableProcess): DisplayableProcess =
```

Comment:
How it is related with checking of `isDisabled` in `FlinkProcessRegistrar`, `PartSubGraphCompiler` and `ProcessValidation`? Is this flag in production code will be used only on frontend side (`DisplayableProcess`) now?